eng/common-tests/get-codeowners/get-codeowners.tests.ps1

```diff
@@ -0,0 +1,50 @@
+Import-Module Pester
+
+BeforeAll {
+  . $PSScriptRoot/../../common/scripts/get-codeowners/get-codeowners-functions.ps1
+
+    $ToolVersion = "1.0.0-dev.20230214.3"
```

Comment:
Do we want to duplicate the version number here?

Reply:
Deduped!